jwt/jwt/jwt.h

```diff
@@ -108,7 +115,7 @@ int setJwtTrace(int toWhat);
 
 int jwtParse(const char *base64Text,
              bool ebcdic,
-             const ICSFP11_HANDLE_T *keyHandle,
+             const JwtContext *self,
```

Comment:
Are we okay with changing a public API? This doesn't seem like a backward compatible change.

Reply:
While we could postpone this until zowe v2 as a breaking change, I don't think its necessary in that I don't know of any users other than us for this feature. I'm ok with it.